FormalConjectures/ErdosProblems/51.lean

```diff
@@ -0,0 +1,46 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 51
+
+*Reference:* [erdosproblems.com/51](https://www.erdosproblems.com/51)
+-/
+
```

Comment:
```suggestion

open Filter
open scoped Nat

```
This way we don't need to write `Filter` below. 
This then allows to use `φ` instead of `Nat.totient` below.